R2DBC Connection issue in webflux-psql job

##### **Overview of the issue**

The [webflux-psql](https://github.com/jhipster/generator-jhipster/runs/2000342378?check_suite_focus=true) job is failing:

```
Error:  Tests run: 6, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 42.385 s <<< FAILURE! - in tech.jhipster.sample.security.DomainUserDetailsServiceIT
Error:  tech.jhipster.sample.security.DomainUserDetailsServiceIT.assertThatEmailIsPrioritizedOverLogin  Time elapsed: 1.691 s  <<< ERROR!
org.springframework.dao.DataAccessResourceFailureException: Failed to obtain R2DBC Connection; nested exception is io.r2dbc.postgresql.PostgresqlConnectionFactory$PostgresConnectionException: Cannot connect to localhost:49154
Caused by: io.r2dbc.postgresql.PostgresqlConnectionFactory$PostgresConnectionException: Cannot connect to localhost:49154
Caused by: io.netty.channel.AbstractChannel$AnnotatedConnectException: finishConnect(..) failed: Connection refused: localhost/0:0:0:0:0:0:0:1:49154
Caused by: java.net.ConnectException: finishConnect(..) failed: Connection refused
```

##### **Motivation for or Use Case**

Tests should pass.

##### **JHipster Version(s)**

`main` branch.

--- a/.github/workflows/webflux.yml
+++ b/.github/workflows/webflux.yml
@@ -138,7 +138,7 @@ jobs:
             environment: prod
             war: 0
             e2e: 1
-            testcontainers: 1
+            testcontainers: 0
           # - app-type: webflux-mysql
           #   entity: sqllight
           #   environment: prod

